src/placement/jump_map_versions.c

```diff
@@ -401,10 +403,35 @@ dom_reset_full(struct pool_domain *dom, uint8_t *dom_bits, uint8_t *tgts_used,
 		clrbit(tgts_used, &dom->do_targets[i] - root->do_targets);
 }
 
+static bool
+dom_tgts_are_avaible(struct pool_domain *dom, uint32_t allow_status, uint32_t allow_version)
```

Comment:
Typo... Minor. dom_tgts_are_avaible --> dom_tgts_are_available.

Reply:
ah, right. will update if it needs to refresh, otherwise in following patch.